.travisci/reportci.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+cat $HOME/.script/annotate.py | python - --name annotate$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION --tool java --input maven.log
+
+cat $HOME/.script/upload.py | python - --name upload$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION --include='*.xml' --framework=junit
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/powerunit/powerunit-extensions-matchers/pullRequest?prid=5559528)